doc/source/whats_new.rst

```diff
@@ -164,6 +166,8 @@ API
    - Use `scipy.signal.welch` instead of matplotlib.psd inside `compute_raw_psd` and `compute_epochs_psd` by `Yousra Bekhti_`
      `Eric Larson_` and `Denis Engemann_`. As a consquence, `Raw.plot_raw_psds` has been deprecated.
 
+   - Deprecated changing the sensor type of channels in `rename_channels` for `define_sensor` by `Teon Brooks`_
```

Comment:
oh, it's because I only searched and replaced over the `mne` package, not the `mne-python` folder